ompi/communicator/communicator.h

```diff
@@ -233,7 +234,7 @@ typedef struct ompi_communicator_t ompi_communicator_t;
  * This is done so we don't end up needing a structure that is
  * incredibly larger than necessary because of the bitness.
  */
-#define PREDEFINED_COMMUNICATOR_PAD (sizeof(void*) * 192)
+#define PREDEFINED_COMMUNICATOR_PAD (sizeof(void*) * 194)
```

Comment:
Ok. Let me double check the math to see if we needed to change it. I don't think we do, but I'd like to make sure.

So the padding works as a compile time check for backwards compatibility on that structure size. You can add pointers to the communicator structure (or something hanging off of it) and if at any point you exceed the padding then you should get a compiler error flagging this as an issue.